h2/src/main/org/h2/mvstore/Cursor.java

```diff
@@ -13,42 +13,76 @@
  * @param <K> the key type
  * @param <V> the value type
  */
-public class Cursor<K, V> implements Iterator<K> {
+public final class Cursor<K, V> implements Iterator<K> {
+    private final K           to;
```

Comment:
I would prefer if the formatting is the same as it was before (not aligned with spaces)

Reply:
No problem...